release-schema.json

```
@@ -532,6 +547,296 @@
+              "MX",
+              "MY",
+              "MZ",
+              "NA",
+              "NC",
+              "NE",
+              "NF",
+              "NG",
+              "NI",
+              "NL",
+              "NO",
+              "NP",
+              "NR",
+              "NU",
+              "NZ",
+              "OM",
+              "PA",
+              "PE",
+              "PF",
+              "PG",
+              "PH",
+              "PK",
+              "PL",
+              "PM",
+              "PN",
+              "PR",
+              "PS",
+              "PT",
+              "PW",
+              "PY",
+              "QA",
+              "RE",
+              "RO",
+              "RS",
+              "RU",
+              "RW",
+              "SA",
+              "SB",
+              "SC",
+              "SD",
+              "SE",
+              "SG",
+              "SH",
+              "SI",
+              "SJ",
+              "SK",
+              "SL",
+              "SM",
+              "SN",
+              "SO",
+              "SR",
+              "SS",
+              "ST",
+              "SV",
+              "SX",
+              "SY",
+              "SZ",
+              "TC",
+              "TD",
+              "TF",
+              "TG",
+              "TH",
+              "TJ",
+              "TK",
+              "TL",
+              "TM",
+              "TN",
+              "TO",
+              "TR",
+              "TT",
+              "TV",
+              "TW",
+              "TZ",
+              "UA",
+              "UG",
+              "UM",
+              "US",
+              "UY",
+              "UZ",
+              "VA",
+              "VC",
+              "VE",
+              "VG",
+              "VI",
+              "VN",
+              "VU",
+              "WF",
+              "WS",
+              "XK",
+              "YE",
+              "YT",
+              "ZA",
+              "ZM",
+              "ZW"
+            ]
+          },
+          "codelist": "country.csv",
+          "openCodelist": false,
+          "minLength": 1
+        },
+        "hasRank": {
+          "title": "Has rank",
+          "description": "Whether this bid is ranked. For more information, see `.rank`.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "rank": {
+          "title": "Rank",
+          "description": "The position of this bid (i.e. whether the bid is ranked first, second, third, etc.) in a design contest, a framework agreement with multiple winners (e.g. cascades), an innovation partnership, a competitive dialogue, or another procedure.",
+          "type": [
+            "integer",
+            "null"
+          ]
+        },
+        "variant": {
+          "title": "Is variant",
+          "description": "Whether the bid is a variant. A variant is a bid that fulfils the buyer's needs in a different way than that expressed in the procurement documents.",
```

American spelling. Align with wording in submissionTerms.

```suggestion
          "description": "Whether the bid is a variant. A variant is a bid that fulfills the buyer's needs differently than as proposed in the procurement documents.",
```